x11-misc/lsw/lsw-0.3.ebuild

```diff
@@ -13,10 +14,12 @@ SLOT="0"
 KEYWORDS="amd64 ~x86"
 
 RDEPEND="x11-libs/libX11"
-DEPEND="${RDEPEND}
-	x11-base/xorg-proto"
+DEPEND="${RDEPEND}"
+BDEPEND="x11-base/xorg-proto"
```

Comment:
sure? `xorg-proto` are just headers for the X data structures, i.e., it has to be for `CHOST`